### ISSUE TITLE
HTML-5 version: team names in room

### ISSUE THREAD

**Issue by axtellclose8** [haxball/haxball-issues#123]
Hello! I think, it will be super to create some new possibilities, For example, what about printing in room team names? Now we have only "red" and "blue", but it'll be much more interesting if person with admin rights will be able to print team name. Especially if it is some champ match...

Thx ))

**Comment by haxballer** [comment]
Good suggestion, mate. I raise the post, because this simple change will make it unnecessary to create alternative scoreboards for streams, which will speed up the process of live work and also can be fun to play in public rooms. It is worth thinking about it, like as slight change, it's fine.

**Comment by Leo-MapCreator** [comment]
Good idea, but people would troll, lmao, "Superteam" Vs "Team of Gays"

**Comment by humpyy** [comment]
What's wrong with Superteam?
